### PR TITLE
strip whitespaces from keyboard input

### PR DESCRIPTION
**Pull request by phil65**
would help me for some of my AutoCompletion stuff. (for some weird reason I have to append a whitespace there, otherwise kodi does not recognize the new label) and shouldnt hurt in any form.

### CODE REVIEW

File: default.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
         keyboard = xbmc.Keyboard( '', __language__(32101), False )
         keyboard.doModal()
         if ( keyboard.isConfirmed() ):
-            searchstring = keyboard.getText()
+            searchstring = keyboard.getText().strip()
     if searchstring:
         import gui
         ui = gui.GUI( "script-globalsearch-main.xml", __cwd__, "Default", searchstring=searchstring )
```
Comment (phil65) [inline default.py]: could also append it here so that it also counts for search strings passed by param.

Comment (ronie) [inline default.py]: yeah, if i need to add it, then do it in a place where it catches all possible occasions.

also, please add a comment-line to explain why the strip() is needed.
(so i don't have to scratch my head a year from now why i ever added that)

**Comment by phil65** [comment]
moved it into GUI class where some other sanitize work is already being done.

**Comment by ronie** [comment]
great, thx!

**Pull request merged**